frontend/src/v5/store/drawings/drawings.sagas.ts

```diff
@@ -59,7 +58,46 @@ export function* fetchDrawingStats({ teamspace, projectId, drawingId }: FetchDra
 	}
 }
 
+export function* fetchCategories({ teamspace, projectId }: FetchCategoriesAction) {
+	try {
+		const categories = yield API.Drawings.fetchCategories(teamspace, projectId);
+		yield put(DrawingsActions.fetchCategoriesSuccess(projectId, categories));
+
+	} catch (error) {
+		yield put(DialogsActions.open('alert', {
+			currentActions: formatMessage({ id: 'drawings.fetchCategories.error', defaultMessage: 'trying to fetch categories' }),
```

Comment:
Should this error perhaps specify 'drawing categories' just so in the future there is no confusion about if an error was from this or risk categories (or any future categories that get added)